src/game/logic/object/objectcreationlist.cpp

```diff
@@ -0,0 +1,61 @@
+#include "objectcreationlist.h"
+
+#ifndef GAME_DLL
+ObjectCreationListStore *g_theObjectCreationListStore = nullptr;
+#endif
+
+// zh: 0x004C1310 wb: 0x0075BB30
+ObjectCreationListStore::~ObjectCreationListStore()
+{
+    for (auto *nugget : m_nuggets) {
+        nugget->Delete_Instance();
+    }
+    m_nuggets.clear();
```

Comment:
Clear call?

Reply:
Makes no difference whether ot not is typed out. It will clear and destruct anyway.

Reply:
I think @tomsons26 is saying this should call `Clear();` but the oclstore doesn't have a Clear function only the ocl has a Clear function. But yes it is pretty pointless as the vector is destroyed anyway after this so it ends up being duplicated anyway. So not sure what you want: 

1. add a Clear function and call it instead of clear?
2.  Remove the clear as it's pointless?

Reply:
I prefer the stl accesses to be buried behind a function if possible, and judging from the functions we know of they did too